### PR TITLE
Test for Loop Law Constraints

### PR DESCRIPTION
**Pull request by tpfau**
A Small test that checks whether looplaw constraints actually lead to elimination of loops.

**I hereby confirm that I have:**

- [X] Tested my code on my own machine
- [X] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [X] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

### CODE REVIEW

File: src/base/solvers/buildLPproblemFromModel.m [hunk 1]
```diff
@@ -0,0 +1,58 @@
+function [ LPproblem ] = buildLPproblemFromModel( model)
+% Builds an COBRA Toolbox LP problem structure from a COBRA Toolbox model structure. 
+%
+% USAGE:
+%
+%    LPproblem = solveCobraLP(model)
```
Comment (laurentheirendt) [inline src/base/solvers/buildLPproblemFromModel.m]: May you change this to:
```
LPproblem  = buildLPproblemFromModel(model)
```

File: src/base/solvers/buildLPproblemFromModel.m [hunk 2]
```diff
@@ -0,0 +1,58 @@
+function [ LPproblem ] = buildLPproblemFromModel( model)
```
Comment (laurentheirendt) [inline src/base/solvers/buildLPproblemFromModel.m]: Formatting

File: src/base/solvers/buildLPproblemFromModel.m [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+%                     * .c - Objective coeff vector
+%                     * .lb - Lower bound vector
+%                     * .ub - Upper bound vector
+%                     * .osense - Objective sense (-1 means maximise (default), 1 means minimise)
+%                     * .csense - Constraint senses, a string containting the constraint sense for
+%                       each row in A ('E', equality, 'G' greater than, 'L' less than).
+
+
+
+LPproblem.A = model.S;
```
Comment (laurentheirendt) [inline src/base/solvers/buildLPproblemFromModel.m]: What if `model` has a field `A`?

File: src/analysis/thermo/thermoFBA/addLoopLawConstraints.m [hunk 4]
```diff
@@ -207,19 +216,19 @@
         -ones(nint,1); % e<
         %-100*ones(nint, 1); % e>
         zeros(linternal,1);];
-
+
```
Comment (laurentheirendt) [inline src/analysis/thermo/thermoFBA/addLoopLawConstraints.m]: May you fix these formatting issues? Thanks.

File: src/base/solvers/buildLPproblemFromModel.m (hunk 3 above)
Comment (tpfau) [reply src/base/solvers/buildLPproblemFromModel.m]: A (according to the definitions) should not be a model field. A is an LP specific field, and thus a field for an LPproblem. If some model contains A, it is already an LPproblem struct..

File: src/analysis/thermo/thermoFBA/addLoopLawConstraints.m (hunk 4 above)
Comment (tpfau) [reply src/analysis/thermo/thermoFBA/addLoopLawConstraints.m]: Not sure, but this is normal matlab indentation of a block. i.e. if its in a command/end block, and automatically indented, all lines in that lbock are set to a specific level.

Comment (laurentheirendt) [reply src/analysis/thermo/thermoFBA/addLoopLawConstraints.m]: These additional indentations just appear weirdly here on Github ... When using [atom](https://atom.io/), these indentations are automatically corrected.

Comment (tpfau) [reply src/analysis/thermo/thermoFBA/addLoopLawConstraints.m]: Well, they are what matlab does for automatic indentation (and I actually think it makes sense, if you have empty lines to keep them on the same indentation level. 
I can remove all the empty lines, but removing the indentation seems odd to me.

**Pull request merged**